.rubocop.yml

```diff
@@ -1,39 +1,13 @@
-Rails:
-  Enabled: true
-inherit_from: .rubocop_todo.yml
-
-Style/HashSyntax:
-  Enabled: true
-  EnforcedStyle: ruby19
-
-# Things deprecated in current ruby API
-Lint/DeprecatedClassMethods:
-  Enabled: true
-
-# Do not compare with nil. Use .nil? instead
-Style/NilComparison:
-  Enabled: true
-
-# Use one empty line between method definitions
-Style/EmptyLineBetweenDefs:
-  Enabled: true
-
-# There should be only one empty line in designated place
-Style/EmptyLines:
-  Enabled: true
-
-# Keep a blank line before and after private.
-Style/EmptyLinesAroundAccessModifier:
-  Enabled: true
-
-# Use hash literal {} instead of Hash.new
-Style/EmptyLiteral:
-  Enabled: true
-
-# Prefer `each` over `for i`
-Style/For:
-  Enabled: true
-
+AllCops:
+  Include:
```

Comment:
Add before this line the following comment: 
`# Include common Ruby source files.`